nifi-commons/nifi-expression-language/pom.xml

```diff
@@ -93,5 +93,15 @@
             <artifactId>commons-text</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.8.1</version>
```

Comment:
can we go to 3.9

Reply:
Bumped and re-tested. AFAICT nothing in the commons-lang3 release suggest a problem.